lib/handler/retry-handler.js

```diff
@@ -159,9 +158,7 @@ class RetryHandler {
     const retryTimeout =
       retryAfterHeader > 0
         ? Math.min(retryAfterHeader, maxTimeout)
-        : Math.min(currentTimeout * timeoutFactor ** counter, maxTimeout)
-
-    state.currentTimeout = retryTimeout
+        : Math.min(minTimeout * timeoutFactor ** counter, maxTimeout)
```

Comment:
if counter starts at 1 should this not be (counter - 1)? This way the first retry uses the `minTimeout` value?

Reply:
right now it would start at double the original minTimeout